impl/src/bitfield.rs

```diff
@@ -246,23 +250,30 @@ impl BitfieldStruct {
         let mut offset = Punctuated::<syn::Expr, Token![+]>::new();
         offset.push(syn::parse_quote! { 0 });
         for (n, field) in self.ast.fields.iter().enumerate() {
-            use crate::ident_ext::IdentExt as _;
-            let field_name = field
+            let field_name: &dyn quote::IdentFragment = match field.ident.as_ref() {
+                Some(field) => field,
+                None => &n,
+            };
+            let getter_name = match field.ident.as_ref() {
+                Some(field_name) => field_name.clone(),
+                None => format_ident!("get_{}", field_name),
+            };
+            let setter_name = format_ident!("set_{}", field_name);
+            let checked_setter_name = format_ident!("set_{}_checked", field_name);
```

Comment:
Unfortunately this reverts our recents changes that changed all getter names to just be their field names. E.g. instead of `fn get_a()` for a field called `a` the getter is now simply with signature `fn a()`.

Reply:
So instead of changing how names are defined simply leave it as it is with the current `master`. We can do a follow-up for some clean-ups.

Reply:
I saw that change and I'm pretty sure this matches the changes made on master, all the test cases pass and reflect the new names, anyway.

Reply:
ah yes you are right! my code review got confused. sorry!